pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1545,6 +1545,20 @@ func (c *Cloud) AttachDisk(diskName KubernetesVolumeID, nodeName types.NodeName,
 		attachResponse, err := c.ec2.AttachVolume(request)
 		if err != nil {
 			attachEnded = true
+			if awsError, ok := err.(awserr.Error); ok {
```

Comment:
the function doesn't have to be on `Cloud` struct btw, it could be a separate function